docs/install.md

```diff
@@ -42,12 +42,14 @@ To update, simply re-run the installation command above.
 
 ### Install a specific version
 
-To install a specific version of Flow CLI, append the version tag to the command (e.g. the command below installs CLI version v0.33.3).
+To install a specific version of Flow CLI newer than v0.42.0, append the version tag to the command (e.g. the command below installs CLI version v0.44.0).
 
 ```sh
-sh -ci "$(curl -fsSL https://raw.githubusercontent.com/onflow/flow-cli/master/install.sh)" -- v0.33.3
+sh -ci "$(curl -fsSL https://raw.githubusercontent.com/onflow/flow-cli/50058df8aa5f999a85cfd7afae6fc2661090078a/install.sh)" -- v0.44.0
```

Comment:
this should be kept `master` for new versions

Reply:
Apologies, forgot to switch it back! I've made the change :)